src/features/content-explorer/content-explorer/ContentExplorerBreadcrumbs.js

```diff
@@ -28,23 +29,25 @@ const ContentExplorerBreadcrumbs = ({
         >
             <IconChevron direction="left" size="6px" color="#333" />
         </Button>
-        <Breadcrumb label={formatMessage(messages.breadcrumb)}>
+        <Breadcrumb label={formatMessage(messages.breadcrumb)} {...breadcrumbProps}>
             {foldersPath.map((folder, i) => (
-                <PlainButton
-                    className="lnk"
-                    key={folder.id}
-                    title={folder.name}
-                    onClick={event => onBreadcrumbClick(i, event)}
-                >
-                    {i === 0 && <IconAllFiles />}
-                    <span>{folder.name}</span>
-                </PlainButton>
+                <div key={folder.id} className="lnk">
```

Comment:
I previously changed this from PlainButton -> div, but to preserve a11y and styling, I opted to wrap the PlainButton within a div.